### ISSUE TITLE
Maryland Death Numbers Do Not Match

### ISSUE THREAD

**Issue by metaconcept** [COVID19Tracking/issues#328]
Maryland death numbers exceed the numbers reported through https://coronavirus.maryland.gov/ .  See historic data screenshots vs data reported in table (https://covidtracking.com/data/state/maryland#historical)

**Comment by tracygs222** [comment]
The discrepancy began with the 4/23/2020 death total, which from that day on, appears to have been calculated as the sum of both the Maryland 'confirmed' and 'probable' death counts. Prior to 4/23/2020, the death numbers reported on this site were consistent with the MD 'confirmed' death count.

**Comment by schmian** [comment]
Hi @metaconcept 

Thanks for reporting this issue!

The CDC issued a guidance on 4/14 that case counts and death counts include confirmed and probable cases [There is more information about the definitions of confirmed and suspected here.](https://www.cdc.gov/coronavirus/2019-ncov/cases-updates/cases-in-us.html)
We have standardized this guideline across all the states that report both numbers. We have also updated Maryland's death counts to include 'probable' deaths beginning on 4/15 when the state began reporting this number. We're in the process of making 'probable' and 'confirmed' deaths more transparent.
